components/Page.js

```diff
@@ -0,0 +1,16 @@
+import NavBar from "./NavBar.js";
+import globalStyles from "../styles/global";
+
+const Page = ({ children }) => {
+  return (
+    <div className="container">
+      <NavBar />
+      {children}
+      <style jsx global>
+        {globalStyles}
+      </style>
+    </div>
```

Comment:
```suggestion
    <div className="container">
      <Head>
        <title>Evolution</title>
        <link rel="icon" href="/favicon.ico" />
      </Head>
      <NavBar />
      {children}
      <style jsx global>
        {globalStyles}
      </style>
    </div>
```

Reply:
so overall the suggested changes are the pages component will have a head but we're still keeping jsx global styling?

Reply:
I'm not sure I understand your question. I think you can click the commit suggestion button to accept these changes and then refactor the index page to be:

```jsx
<Page>
   <Header />
   <Footer />
</Page>
```

Reply:
addressed in the last commit. Thanks for clarifying!